helper/cachehelper/cachehelper.go

```diff
@@ -0,0 +1,75 @@
+package cachehelper
+
+import (
+	"sync"
+	"time"
+)
+
+// Cache represents the interface for a simple cache.
+type Cache interface {
```

Comment:
Since we know that these will be string values, I think this cache doesn't need to be interface valued, right? Unless I'm missing something and there could be other types values?

Reply:
I thought the cache could be generic enough that we could reuse if required?

Reply:
we'll alter it when we get there I think. :) We might never reuse it. :)

If we would, it's very easy to change it to interface.

Reply:
Sure thing :+1: